### PR TITLE
Fix check online in DEBIAN/postinst from ping to curl

### PR DESCRIPTION
**Pull request by adeepn**
In some virtual machines ping sockets may be disabled, but curl works good.

### CODE REVIEW

File: homeassistant-supervised/DEBIAN/postinst [hunk 1]
```diff
@@ -65,7 +65,7 @@ info "Restarting docker service"
 systemctl restart "${SERVICE_DOCKER}"
 
 # Check network connection
-while ! ping -c 1 -W 1 ${URL_CHECK_ONLINE}; do
+while ! curl -q ${URL_CHECK_ONLINE} >/dev/null 2>&1 ; do
```
Comment (agners) [inline homeassistant-supervised/DEBIAN/postinst:68]: `URL_CHECK_ONLINE` has no protocol prefixed. It seems that curl automatically applies a protocol, but ideally we are explicit here:

```suggestion
while ! curl -q "https://${URL_CHECK_ONLINE}" >/dev/null 2>&1 ; do
```

**Review by agners — CHANGES_REQUESTED** [review]

File: homeassistant-supervised/DEBIAN/postinst (hunk 1 above)
Comment (agners) [reply homeassistant-supervised/DEBIAN/postinst:68]: Actually, curling `https://checkonline.home-assistant.io/` is slow for me. Ideally we curl `https://checkonline.home-assistant.io/online.txt` directly here. Can you adjust `URL_CHECK_ONLINE` accordingly?

Comment (adeepn) [reply homeassistant-supervised/DEBIAN/postinst:68]: > Actually, curling `https://checkonline.home-assistant.io/` is slow for me. Ideally we curl `https://checkonline.home-assistant.io/online.txt` directly here. Can you adjust `URL_CHECK_ONLINE` accordingly?

better to move protocol to URL_CHECK_ONLINE too.

Comment (agners) [reply homeassistant-supervised/DEBIAN/postinst:68]: Not sure why exactly, but at least here when I curl `https://checkonline.home-assistant.io/`, it is rather slow. Curling `https://checkonline.home-assistant.io/online.txt` is fast. And that is also the location used by the Supervisor online check, so I'd go for the full URL with file.

**Review by agners — CHANGES_REQUESTED** [review]

File: homeassistant-supervised/DEBIAN/postinst (hunk 1 above)
Comment (adeepn) [reply homeassistant-supervised/DEBIAN/postinst:68]: sorry, my fault on replace url :)

**Pull request merged**